src/react/boosts/BoostForm.tsx

```diff
@@ -1,38 +1,147 @@
 import * as React from "react";
 import styled, { StyledComponentClass } from "styled-components";
-import { colors, fonts, mediaQueries, Button, buttonSizes, CurrencyInput, TextInput, TextareaInput } from "@joincivil/components";
+import { colors, fonts, mediaQueries, Button, buttonSizes, CurrencyInput, InputIcon, TextInput, TextareaInput, QuestionToolTip, defaultNewsroomImgUrl } from "@joincivil/components";
 import { Mutation, MutationFunc } from "react-apollo";
 import { boostMutation } from "./queries";
 import { BoostData } from "./types";
+import { BoostImg, BoostWrapper, BoostWrapperFullWidthHr, BoostFormTitle, BoostDescriptionTable, BoostPayFormTitle, BoostSmallPrint } from "./BoostStyledComponents";
 
-const BoostWrapper = styled.div`
-  border: 1px solid ${colors.accent.CIVIL_GRAY_4};
-  font-family: ${fonts.SANS_SERIF};
-  font-weight: normal;
+const PageWrapper = styled.div`
+  color: ${colors.primary.CIVIL_GRAY_0};
   font-size: 14px;
-  margin: 0 auto 50px;
-  padding: 30px;
-  position: relative;
+  font-weight: normal;
+  margin: auto;
+  max-width: 880px;
+  padding: 50px 20px 20px;
 
-  ${mediaQueries.MOBILE} {
-    padding: 20px;
+  p {
+    margin-top: 15px;
+  }
+
+  a {
+    color: ${colors.accent.CIVIL_PURPLE};
+  }
+
+  input[disabled] {
+    background: white;
+    color: ${colors.primary.CIVIL_GRAY_2};
+  }
+
+  &:after {
+    content: "";
+    display: table;
+    clear: both;
   }
 `;
-const Title = styled.h3`
-`;
-const FormTitle = styled.h4`
+
+const Title = styled.div`
+  font-size: 24px;
+  font-weight: 900;
+  line-height: 33px;
 `;
 const Error = styled.span`
   color: red;
 `;
 
+const NewsroomDetailRow = styled.div`
+  display: flex;
+  justify-content: space-between;
+  ${mediaQueries.MOBILE} {
+    display: block;
+  }
+`;
+const NewsroomDetailCell = styled.div`
+  flex-grow: 1;
+  margin-right: 24px;
+  &:last-child {
+    margin-right: 0;
+  }
+  ${mediaQueries.MOBILE} {
+    margin-right: 0;
+  }
+`;
+
+const ItemsTableWrapper = styled(BoostDescriptionTable)`
+  table td {
+    padding-top: 0;
+    padding-bottom: 0;
+    &:last-child {
+      padding-right: 0;
+    }
+  }
+  tfoot tr:first-child td {
+    padding-top: 36px;
+  }
+`;
+const ItemLink = styled.a`
+  font-size: 16px;
+  font-weight: 500;
+`;
+const RemoveItem = styled(ItemLink)`
+  position: relative;
+  top: -6px;
+`;
+const ItemNameInput = styled(TextInput)`
+  max-width: 420px;
+`;
+const ItemCostHeaderCell = styled.th`
+  && {
```

Comment:
Does this work? usually it would be `& + &`

Reply:
`&&` is different, it's a way to increase specificity without doing `!important`. E.g. the selector ends up as `.classname.classname` which ups specificity without changing what it selects. In this case I had to override some other `th` style that was showing up.